package/src/CodeScanner.ts

```diff
@@ -36,6 +50,13 @@ export interface Code {
     width: number
     height: number
   }
+  /**
+   * The location of each corner relative to the Camera Preview (in dp).
+   */
+  corners?: {
```

Comment:
@metrix-hu 
It might be helpful to note that for `ios` one has to multiply points x and y by the frame width and frame height respectively. 
If you're looking to show the corners, you don't need to do anything for android, but for ios, it would need to be transformed where:

```
x = point.x * frameWidth
y = point.y * frameHeight
```

Reply:
@only1chi Well,not anymore I just fixed that in one of my latest commits.